openhtf/plugs/usb/__init__.py

```diff
@@ -86,32 +86,48 @@ def _open_usb_handle(**kwargs):
 class FastbootPlug(plugs.BasePlug):
   """Plug that provides fastboot."""
 
-  def __new__(cls):
-    device = fastboot_device.FastbootDevice.connect(
+  def __init__(self):
+    self._device = fastboot_device.FastbootDevice.connect(
         _open_usb_handle(
             interface_class=fastboot_device.CLASS,
             interface_subclass=fastboot_device.SUBCLASS,
             interface_protocol=fastboot_device.PROTOCOL))
-    device.tearDown = device.close  # pylint: disable=invalid-name
-    return device
+
+  def tearDown(self):
+    self._device.close()
+
+  def __getattr__(self, attr):
+    """Forward other attributes to the device."""
+    # Use the class version of logger.
+    if attr == 'logger':
+      return self.__getattribute__(attr)
```

Comment:
My original implementation for a similar plug required this code to pass the PlugManager check pre #466.

This is no longer necessary, so it will now forward any unknown attribute to the device class.